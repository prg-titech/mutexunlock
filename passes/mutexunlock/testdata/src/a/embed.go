package main

import "sync"

type T struct {
	sync.Mutex
}

func (t *T) F() {
	t.Lock()
} // want "missing unlock"
