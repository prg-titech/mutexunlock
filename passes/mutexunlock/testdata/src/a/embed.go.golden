package main

import "sync"

type T struct {
	sync.Mutex
}

func (t *T) F() {
	t.Lock()
	t.Unlock()
} // want "missing unlock"
